### DISCUSSION TITLE
Managing submissions and peer review + OJS/PKP

### DISCUSSION THREAD

**Discussion by emmaa-i-ae** [knowledgefutures/pubpub#3120] [Support]
I am in the process of starting a scholarly journal and have been exploring different options for hosting, management, and publishing. 

PubPub is the only platform I have come across that actually has an aesthetically pleasing and clean interface. I really like the ease of using it so far. My main concern is the lack of support for managing submissions and peer review.

Will there be better support for peer review in the foreseeable future? 

Meanwhile, I am considering using OJS/PKP (installed on a subdomain) to manage the journal's workflow (submissions, peer review, copyediting, etc) and then using PubPub (on the main domain) to publish issues and be the public face of the journal. I know it's possible to customise OJS/PKP but it's a lot of work and a little over my head, and I really like PubPub!

Does anyone have experience of using PubPub alongside OJS/PKP? Or, am I making things too complicated?

**Comment by eastofesten** [comment]
Hello! We're excited to hear about your journal. Yes, we are working on better support for peer review in the near future, and it's not uncommon for communities to use PubPub alongside other tools for managing workflows. You may want to reach out to Partnerships team [via this form](https://www.pubpub.org/community-services) to discuss your journal and its needs re: PubPub.

**Comment by aimeediehl** [comment]
@emmaa-i-ae, the University of Vermont Press uses exactly the process you describe—editorial workflow on OJS and publishing on PubPub. Contact me at press@uvm.edu if you'd like to chat!
